logger.go

```diff
@@ -188,3 +201,22 @@ func (log Logger) log(evt *zerolog.Event, message string, fields ...Data) {
 
 	evt.Int64("nanoseconds", zerolog.TimestampFunc().UnixNano()).Msg(message)
 }
+
+func getContainerId() (string, error) {
+	content, err := ioutil.ReadFile("/proc/1/cpuset")
+	if err != nil {
+		return "", err
+	}
+
+	// The format is /namespace/subNamespace/containerId
+	// Split the content of the file
+	parts := strings.Split(string(content), "/")
+
+	// Pull the last element form the split
+	id := parts[len(parts)-1]
+
+	// Remove whitespace (probably just newlines)
+	clean_id := strings.TrimSpace(id)
+
+	return clean_id, nil
```

Comment:
nit: no need for another var
```suggestion
	return strings.TrimSpace(id), nil
```